src/modules/categories/components/category/category.jsx

```diff
@@ -41,7 +41,7 @@ export default class Category extends Component {
     const s = this.state
 
     const isNullCategory = p.category === 'null-category' && p.popularity === 0
-    const roundedPop = new BigNumber(p.popularity).integerValue(BigNumber.ROUND_HALF_EVEN)
+    const roundedPop = new BigNumber(p.popularity.toString()).integerValue(BigNumber.ROUND_HALF_EVEN)
```

Comment:
added this so UI would not blow up. It's on BigNumber follow PR.

Reply:
This change has already been made on seadragon.

Reply:
To clarify: the equivalent update was made in augur-node, so that the values that arrive from augur-node are already strings.  It doesn't do any harm to have `.toString()` here, although it isn't needed.

Reply:
(It does add some confusion though, since it implies that `popularity` could be something besides a string -- it can't be -- so I suggest removing the `.toString()`.)

Reply:
actually this change is already merged up to seadragon, need to rebase this branch.